### DISCUSSION TITLE
PTZ URL for motioneyeos

### DISCUSSION THREAD

**Discussion by ronstep** [roleoroleo/sonoff-hack#88] [Q&A] [answered]
Does anyone know the url protocol for pan/tilts

ie http://192.168.100.114:9595/api/tilt/

I'm trying to use action buttons on motioneye 

thanks

**Comment by roleoroleo** [comment, marked as answer]
Check the wiki:
https://github.com/roleoroleo/sonoff-hack/wiki/Web-services-description

**Reply by ronstep** [reply]
My bad, thanks for prompt reply

**Reply by ronstep** [reply]
Quick question should this work from browser as nothing happens to camera and I get this response 

{
"error":"false"
}

regards
